packages/ui-drawer-layout/src/DrawerLayout/DrawerTray/index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/require-default-props */
```

Comment:
Is this in the wrong place? We should not exclude whole files from linting if possible

Reply:
Fixed in all of the files which was affected.